test/unit/range/views/get_test.cpp

```diff
@@ -74,80 +59,53 @@ TEST(view_get, advanced)
                                                    {'C'_dna4, seqan3::mask::UNMASKED},
                                                    {'G'_dna4, seqan3::mask::MASKED},
                                                    {'T'_dna4, seqan3::mask::UNMASKED}};
-    std::vector<seqan3::masked<seqan3::dna4>> functor0 = seqan3::views::get<0>(t) | seqan3::views::to<std::vector>;
-    EXPECT_EQ(cmp0, functor0);
 
-    std::vector<seqan3::phred42> cmp1{seqan3::phred42{0}, seqan3::phred42{1}, seqan3::phred42{2}, seqan3::phred42{3}};
-    std::vector<seqan3::phred42> functor1 = seqan3::views::get<1>(t) | seqan3::views::to<std::vector>;
-    EXPECT_EQ(cmp1, functor1);
+    EXPECT_RANGE_EQ(cmp0, seqan3::views::get<0>(t));
```

Comment:
As there is no `cmp00` anymore you could rename `cmp0` to `cmp` or maybe maybe use a more explanatory name as `functor_compared`.

Reply:
I renamed the stuff in the get test.